code/utils.py

```diff
@@ -31,4 +31,12 @@ def required_parameters_provided(parameters, keys, message="Required parameter n
 
 def mask_parameter(parameter):
     print(f"::add-mask::{parameter}")
+    
+def get_events_list(events_to_subscribe):
```

Comment:
what happens when user provide empty list?

second , what user will do to provide all events something like "ALL"?